services/minespace-web/src/components/common/RenderSelect.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
```

Comment:
Why do we need this?

Reply:
When I started making changes in this file, it made the linter starting checking it again, which caused some "no prop type defined" for a few of the props which I hadn't bothered to fix yet.